index.html

```diff
@@ -1,215 +1,243 @@
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+          <a class="hamburger">
+            <i class="fa fa-solid fa-bars"></i>
+          </a>
+          <ul class="menu-bar">
+            <li class="nav"><a href="#Portfolio">Portfolio</a></li>
+            <li class="nav"><a href="#aboutme">About</a></li>
+            <li class="nav"><a href="#contact-me">Contact</a></li>
+          </ul>
+        </nav>
 
-            <div class="card port-four">
-                <div class="profile order-1"><img src="image-source/Snapshoot Portfolio (3).png" alt="Snapshoot"></div>
-                <div class="card-port-description order-2">
-                    <h3>Multi-Post Stories</h3>
-                    <div class="profile-description">
-                        <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>2015</p>
-                    </div>
-                    <div class="paragraph-description">
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <ul class="lang">
-                        <li>html</li>
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
-        </section>
-        <section id="About" class="About-me About-major">
-            <div class="one">
-                <h2>About <br> Myself</h2>
-                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                <h3 class="let">LET'S CONNECT</h3>
-                <ul class="nav-links">
-                    <li><a href="https://twitter.com/Ezechukwuchibu2"><img src="image-source/twitter.svg" alt="twitter"></a></li>
-                    <li><a href="https://www.linkedin.com/in/ezechukwu-chibuike/"><img src="image-source/Linkedin icon.svg" alt="Linkedin"></a></li>
-                    <li><a href="#"><img src="image-source/medium.svg" alt="Vector-1"></a></li>
-                    <li><a href="https://github.com/Moses-chibuike"><img src="image-source/github.svg" alt="github"></a></li>
-                    <li><a href="#"><img src="image-source/angellist.svg" alt="Vector"></a></li>
-                </ul>
-                <div class="section-two-desc">
-                    <button class="btn" type="button">Get my resume</button>
-                </div>
-            </div>
```

Comment:
I think you have mistakenly taken away your portfolio section and it is a requirement that when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed😉. kindly add the portfolio section back so that when a user clicks the portfolio link on the mobile menu, it takes him to the portfolio section😉.
![Screenshot from 2023-01-30 20-37-54](https://user-images.githubusercontent.com/64643995/215578692-abaddd79-99eb-440d-8514-d6ef70012e8b.png)

your project
![Screenshot from 2023-01-30 20-27-43](https://user-images.githubusercontent.com/64643995/215581276-6b8cd1d2-c8cf-4f4f-853c-dd9b6305f72e.png)

Reply:
Thanks for the review @Ibizugbe